content/docs/howto/app-monitor.md

```diff
@@ -0,0 +1,267 @@
+
+The buildpack supports the Java language.
+
+### Connecting with YourKit
+
+YourKit, which runs on your local computer, must connect to a listening port within your container. When starting an application with debugging enabled, a port must be published. 
+
+To publish the port in Docker, use the following command:
+
+{{< code/copyable >}}
+$ docker run --publish <LOCAL_PORT>:<REMOTE_PORT> ...
+{{< /code/copyable >}}
+
+The `REMOTE_PORT` should match the `port` configuration for the application (`10001` by default).  The `LOCAL_PORT` can be any open port on your computer, but typically matches the `REMOTE_PORT` where possible.
+
+Once the port has been published, your YourKit Profiler, running on your local computer, should connect to `localhost:<LOCAL_PORT>` for profiling.
+
+![YourKit Configuration](https://raw.githubusercontent.com/paketo-buildpacks/yourkit/main/yourkit.png)
+
+## Examples
+
+The following examples show how to use Paketo APM buildpacks with your applications. The examples are broken down into two different types, those that require a binding and those that require an environment variable. Check the buildpack for your APM tool of choice to see if it requires a binding or an environment variable and follow the appropriate instructions.
+
+### Example Prerequisites
+
+Since all of the Paketo application monitoring buildpacks support the Java programming languages we'll use the [Java sample applications][samples/java].
+
+To run the examples, clone the repository and change to the Java sample application directory.
+
+{{< code/copyable >}}
+git clone https://github.com/paketo-buildpacks/samples
+cd samples/java
+{{< /code/copyable >}}
+
+Both of the examples assume that the [Paketo Base builder][base builder] is the default builder. If you haven't already, set it as the default now.
+
+{{< code/copyable >}}
+pack config default-builder paketobuildpacks/builder:base
+{{< /code/copyable >}}
+
+### Example with a Binding
+
+Many of the APM tools require a binding to trigger the tools to be installed and to pass required credentials at runtime. This example uses Azure Application Insights which is one such APM tool.
+
+1. `cd application-insights`.
+2. Build an image with the Azure Application Insights Java Agent:
+
+    {{< code/copyable >}}
+pack build samples/java --volume "$(pwd)/binding:/platform/bindings/application-insights"
+{{< /code/copyable >}}
+
+3. To connect to Azure Application Insights at runtime a valid [Instrumentation Key][azure application insights instrumentation key] is required.
+
+    {{< code/copyable >}}
+echo "<Instrumentation Key>" > binding/InstrumentationKey
+docker run --rm --tty --publish 8080:8080 \
+  --env SERVICE_BINDING_ROOT=/bindings \
```

Comment:
Not a big deal if you follow it properly but can this be shown without overriding the SERVICE_BINDING_ROOT? I feel like bindings are tricky as it is

Reply:
Yes, if you just use `/platform/bindings` it should work w/out that env variable.